src_code/src/utils/services/helper_functions/db_helper_functions/DBHelperFunctions.js

```diff
@@ -0,0 +1,47 @@
+async function GetUserInfoBasedOnEmailAddress(db, emailAddress){ 
+    try {
+        /** Initailize an user object query from database */
+        const user = await db.select(
+            "PK_USER_ID",
+            "FK_CDE_ROLE", 
+            "FIRST_NAME",
+            "MIDDLE_NAME",
+            "LAST_NAME",
+            "STUDENT_ID",
+            "EMAIL_ADDRESS",
+            "TAG_ID",
+        ).from("user_info").where("EMAIL_ADDRESS", "=", emailAddress.toLowerCase().trim()).first();
+
+        /** If emailAddress is not exists, then the user is not exist */
+        if(!user) {
+            return null;
+        }
+
+        /** Get user role description */
+        const userRole = await db.select("DESC_ROLE").from("user_role").where("PK_CDE_ROLE", "=", user.FK_CDE_ROLE).first();  
```

Comment:
Same as above, as we discussed in the database schema, we will not using the user_role table anymore, so we won't need to get the code role from the user_role table.